tracpro/polls/models.py

```diff
@@ -71,8 +71,13 @@ def set_active_for_org(self, org, uuids):
     def sync(self, org):
         """Update the org's Polls from RapidPro."""
         # Retrieve current Polls known to RapidPro.
-        temba_polls = get_client(org).get_flows()
-        temba_polls = {p.uuid: p for p in temba_polls}
+        temba_polls_result = get_client(org).get_flows()
+
+        # Filter out polls with names starting with 'Single Message'
+        temba_polls = {}
+        for poll in temba_polls_result:
+            if poll.name.lower()[:14] != 'single message':
```

Comment:
poll isn't a Django queryset object but rather a tembaclient Flow object, so startswith isn't available. 

I could remove the lower() if we don't think it's necessary, that's one less string operation...